igor_src/Aligner.h

```diff
@@ -100,10 +100,13 @@ class Aligner {
 	virtual ~Aligner();
 
 	std::forward_list<Alignment_data> align_seq(std::string , double , bool , int , int);
+	std::forward_list<Alignment_data> align_seq(std::string , double , bool , std::unordered_map<std::string,std::pair<int,int>>, bool);
```

Comment:
There's quite a lot of code duplication with the function right above. In the end this one is just more general, and   
```
std::forward_list<Alignment_data> align_seq(std::string , double , bool , int , int);
```
should call 
```
std::forward_list<Alignment_data> align_seq(std::string , double , bool , std::unordered_map<std::string,std::pair<int,int>>, bool);
```
under the hood!